### ISSUE TITLE
Disable funlen for *_test.go files

### ISSUE THREAD

**Issue by renkam** [ultraware/funlen#5]
Is there any option to disable this linter for all test files?
Often table tests have input tables very long and it is not a problem for reading test function.

**Comment by NiseVoid** [comment]
Hello,

When we wish to disable linters in tests we add a rule to `exclude-rules` under `issues` in the `.golangci.yml` like this:

```yaml
issues:
  exclude-rules:
    - path: _test.go
      linters:
        - funlen
```

Would this be an acceptable solution?

**Comment by renkam** [comment]
This would probably solve the case.
It would be nice to have such comment in readme.

Is there option to apply this per func name like `Test*`?

**Comment by NiseVoid** [comment]
Sorry for the late reply.

I'm not completely sure if there's a pretty way to do that. All I could find is using the source regex, something like this:

```
issues:
  exclude-rules:
    - linters:
        - funlen
      source: "^func Test"
```

This would disable the rule on any line that matches the regex. This would work because the error funlen gives is on the line where the function starts, however it wouldn't work for most other linters.

**Comment by robinknaapen** [comment]
@renkam 

First if all sorry for the late reply.

Was the snippet provided by @NiseVoid sufficient?
If so would it be fine to close the issue?

**Comment by martialblog** [comment]
Maybe there could be a small hint in the README, with reference to the golangci config

**Comment by robinknaapen** [comment]
Fixed in #17
